File: 0027-remove-element/0027-remove-element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        ans = []
        for num in nums:
            if num != val:
                ans.append(num)
        n = len(ans)
        for i in range(n):
            nums[i] = ans[i]

        return n

File: 0027-remove-element/0027-remove-element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int cont = 0;
        vector<int> ans;
        for(int i=0; i<nums.size(); i++) {
            if(nums[i] != val) {
                cont++;
                ans.push_back(nums[i]);
            }
        }

        for(int i=0; i<cont; i++) {
            nums[i] = ans[i];
        }

        return cont;
    }
};